README/README_pt.md

```diff
@@ -30,21 +30,56 @@ const sisp = new Sisp({ posID, posAutCode, url });
 ```
 ## Gerar formulário de solicitação de pagamento
 ```js
-sisp.generatePaymentRequestForm(referenceId, total, webhookUrl);
+sisp.generatePaymentRequestForm(referenceId, total, webhookUrl, paymentRequestFormOptions?);
 ```
 Gera e retorna um formulário HTML que pode ser usado para processar pagamentos.
 
 * `referenceId`: Client-generated payment reference ID. Este valor é enviado através de uma requisição  `POST` para o  `webhookUrl` e permite ao cliente correlacionar a solicitação de pagamento e a resposta.
 * `total`: Valor do pagamento (em CVE).
 * `webhookUrl`: A url para onde o SISP deve enviar a resposta de pagamento. Você deve esperar uma requisição `POST` com informações de pagamento no corpo. Veja a [documentação](https://www.vinti4.cv/documentation.aspx?id=682) do SISP para mais informações.
+* `paymentRequestFormOptions`: Opções para especificar o tipo de formulário de pagamento a ser gerado.
+
+    ```
+    const paymentRequestFormOptions = {
+        transactionCode,
+        entityCode,
+        referenceNumber
+    }
+    ```
+
+    * `transactionCode`: O tipo de transação a ser feito. O valor padrão é 1.
+        - Códigos de transação possíveis
+
+            | Transação: `string`    | transactionCode: `string` |
+            |------------------------|---------------------------|
+            | Compra                 |             1             |
+            | Pagamento de serviço   |             2             |
+            | Recarga de telefone    |             3             |
+    * `entityCode`: O código da entidade que receberá o pagamento. Deve estar presente quando o transactionCode for igual a 2 ou 3.
+    * `referenceNumber`: O número de referência do serviço a ser pago. Quando transactionCode for igual a 2 o número de referência é o número de referência da fatura a ser paga, quando transactionCode for igual a 3, o número de referência deve ser o número de telefone a ser recarregado. Deve estar presente quando o transactionCode for igual a 2 ou 3.
+
+### Exemplo 1
+```js
+const referenceId = 'abc-123';
+const total = 1200;
+const webhookUrl = 'https://samba.chuva.io/webhooks/sisp-payment';
 
-### Exemplo
+const htmlForm = sisp.generatePaymentRequestForm(referenceId, total, webhookUrl, paymentRequestFormOptions);
```

Comment:
@luis737 In this example you don't need to specify the paymentRequestFormOptions

Reply:
Thanks @FredSoares good catch